golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -23,14 +24,82 @@ type StreamCell struct {
 	Values      []string `json:"values"`
 }
 
+type Change struct {
+	Path         string
+	Value        string
+	OldValue     string
```

Comment:
Comment that the old value is what's in the keeper's store - not what's in a previous uncommitted change.